doc/relations-handling.md

```diff
@@ -18,6 +18,10 @@ Nelmio\Entity\Group:
         owner: '@user1'
 ```
 
+**Warning:** References (e.g. here `group1`) should always be lowercase and not containing any of the following special
+characters: `{`, `}`, `-`, `>`, `'`. They all are reserved characters, they may be used but will have a special meaning
```

Comment:
To be changed: can be composed of letters + digits + `.`, `_`, `/` and must contain at least one letter